modules/webapp/src/main/elm/Comp/ItemDetail/SingleAttachment.elm

```diff
@@ -112,6 +107,26 @@ attachHeader texts settings model _ attach =
         multiAttach =
             List.length model.item.attachments > 1
 
+        selectPossible =
+            multiAttach && model.attachMenuOpen
+
+        selectView =
+            case model.viewMode of
+                SelectView _ ->
+                    True
+                _ ->
```

Comment:
nitpicking: I usually list all possible values here, so that the compiler yells at me should I add another branch some day.

Reply:
Adjusted with https://github.com/eikek/docspell/pull/768/commits/5faf0e5a0eec09a2f4021726f68d23202e521ccf